### DISCUSSION TITLE
Some of my w.i.p. levelset

### DISCUSSION THREAD

**Discussion by DonAlgo** [Neverball/neverball#307] [Show and tell]
Another sneak peak from my levelset in developement (this is a _bonus level_ btw)

![image](https://user-images.githubusercontent.com/82674433/196558715-c16acf9c-620c-4eae-ab12-abac428d73fe.png)

![image](https://user-images.githubusercontent.com/82674433/196558532-a1941b51-607d-4eb6-837f-24b5f9bd70ce.png)

**Comment by DonAlgo** [comment]
![image](https://user-images.githubusercontent.com/82674433/196996032-b6081e92-cc62-46e4-9e9b-404cefdb12b8.png)
![image](https://user-images.githubusercontent.com/82674433/196996066-d866db18-8ea9-4287-8366-754f1b90b233.png)

more

**Reply by DonAlgo** [reply]
![image](https://user-images.githubusercontent.com/82674433/197429502-5c0bc0b1-b9eb-4fc4-b28f-d21257581932.png)
![image](https://user-images.githubusercontent.com/82674433/197429542-6827640d-f1a7-45d6-aea0-20bf0ef28b82.png)
yet another one

**Reply by Adacrop-The-Great** [reply]
this looks epic!

**Reply by DonAlgo** [reply]
thanks!!!

**Comment by rlk** [comment]
These look really great.

The giant ball floor design is beautiful, and really a nice idea. Love it.

Maybe make the white material reflective so we can see the reflection of the sky in it.

**Reply by DonAlgo** [reply]
Wow, thank you so much!!!

And yeah, I tried using mirror material but it isn't 'white' enough, so I think I'll make a custom texture
